### ISSUE TITLE
Add Boom and MBF compatibility modes

### ISSUE THREAD

**Issue by ioan-chera** [team-eternity/eternity#258]
Just like -vanilla.

**Comment by Altazimuth** [comment]
Good idea, though I maintain that -complevel is still handy for those already familiar with it.
Not sure what a good name for Boom 2.02 would be, or Doom 1.9.

**Comment by ioan-chera** [comment]
Well I was thinking of command line parameters like -mbf, -boom (alias of -boom203), maybe even -boom202. Anything to set demo_version and lock some gameplay settings to that. And they, along with -vanilla and demo playback, will be the only ways to change demo_version.

**Comment by Gaerzi** [comment]
What is Boom 203? AFAIK there's just Boom 201 (PrBoom's complevel 8, which nobody ever uses) and Boom202 (PrBoom's complevel 9, which everyone uses).

I can see 2 (-commercial, -vanilla), 3 (-ultimate), 4 (-final), 9 (-boom), and 11 (-mbf) being potentially useful. The others are far too niche to be of interest outside of PrBoom+ IMO. And even -3 and -4 are most probably overkill; when people make new "oldschool" maps they only ever target complevels 2, 9 and 11.
